src/components/LoadingSpinner/LoadingSpinner.scss

```diff
@@ -38,12 +48,22 @@
     }
 
     &__path {
+        stroke: var(--color-primary-50);
         stroke-dasharray: 1, 150;
         stroke-dashoffset: 0;
         stroke-linecap: round;
         animation: dash 1.5s ease-in-out infinite;
         transition: stroke var(--transition-duration);
-        stroke: var(--color-brand-50);
+        @each $context in $contexts {
+            &--context_#{ $context } {
+                $mappedContext: map.get($contextMap, $context);
+                @if ($mappedContext == 'whiteSpinner') {
+                    stroke: var(--color-background);
```

Comment:
please check what is used in figma. `--color-background` is a "left over" from previous tokes we had, we should use the new one when migrating things.